common/models/user.js

```diff
@@ -558,6 +564,12 @@ module.exports = function(User) {
       return cb.promise;
     }
 
+    if ((options.password) && (options.password).length > MAXLENGTH) {
+      var err = new Error(g.f('Password cannot exceed %j characters', MAXLENGTH));
```

Comment:
%j -> %d (assuming the globalization API is the same as util.format -- please verify with @Setogit)